client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -147,25 +148,29 @@ class Slideshow extends Component {
 							</li>
 						) ) }
 					</ul>
-					<div
-						className="wp-block-jetpack-slideshow_pagination swiper-pagination swiper-pagination-white"
-						ref={ this.paginationRef }
-					/>
-					<button
+					<a
 						className="wp-block-jetpack-slideshow_button-prev swiper-button-prev swiper-button-white"
 						ref={ this.btnPrevRef }
+						role="button"
 					/>
-					<button
+					<a
 						className="wp-block-jetpack-slideshow_button-next swiper-button-next swiper-button-white"
 						ref={ this.btnNextRef }
+						role="button"
 					/>
-					<button
+					<a
 						aria-label={ __( 'Pause Slideshow' ) }
```

Comment:
We still have a localized string in the serialized component, right?

Reply:
Yes you're right, and in fact that is present on `master`, separate from any of the changes to `<button>` handling: https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/extensions/slideshow/slideshow.js#L163

I'm inclined to simply change that text to plain English for V1. There is already other non-localized a11y text in the block as described here: https://github.com/Automattic/wp-calypso/issues/31124. Since we'll have to address all of these localization concerns in a future iteration, there's no point pushing for a half-solution right now. Agree?

Reply:
I'm not sure what's a worse offense: shipping unlocalized a11y UI, or dropping the a11y texts entirely.

I agree that one of those approaches should be taken before the block is released, whichever is the lesser offense.

Reply:
`dropping the a11y texts entirely` will affect all a11y-reliant customers, while `shipping unlocalized a11y UI` will affect all non-English speaking a11y-reliant customers. If our goal is to adversely affect as few customers as possible, then `shipping unlocalized a11y UI` is the better way forward, so I'll go that way.

Reply:
Sounds good.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/31175/commits/ec89684d16f902d11d9247f8902f447883cece1e